srv/GetObjectsOnTable.srv

```diff
@@ -0,0 +1,3 @@
+sensor_msgs/PointCloud2 point_cloud
+---
+vision_msgs/BoundingBox3DArray detected_objects
```

Comment:
I am a little confused by the two services, the TablePlaneExtractor service should probably only return the plane, or the plane and a plane bounding box?

On the other hand, the GetObjectOnTable could return the extracted clouds? That relatively trivial from the bounding box and not returning it avoid sending point clouds around, but it might be more convenient depending on the pipeline. Maybe both could be made available?

Reply:
Implemented.
TablePlaneExtractor now returns plane and plane bounding box.
GetObjectOnTable now has two services: one that returns bounding boxes and one that returns pointclouds.